homeassistant/components/reolink/strings.json

```diff
@@ -19,7 +19,7 @@
     },
     "error": {
       "api_error": "API error occurred",
-      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%], check the IP address of the camera and see the troubleshooting steps in the documentation",
```

Comment:
We try to avoid concatenating translation strings using references as different languages may have different requirements in the sentence structure for different clauses.

It's better to write a whole new string if we don't want to just use the reference string.

Although I think these tips are not needed. Checking the docs is standard.